src/main/java/com/ajou/hertz/domain/user/controller/UserController.java

```diff
@@ -117,6 +122,24 @@ public ResponseEntity<UserResponse> signUpV1(
 			.body(UserResponse.from(userCreated));
 	}
 
+	@Operation(
+		summary = "프로필 이미지 변경",
+		description = "프로필 이미지를 변경합니다.",
+		security = @SecurityRequirement(name = "access-token")
+	)
+	@PutMapping(
+		value = "/me/profile-images",
+		headers = API_VERSION_HEADER_NAME + "=" + 1,
+		consumes = MediaType.MULTIPART_FORM_DATA_VALUE
+	)
```

Comment:
받고자 하는 media type을 multipart/form-data 제한할 필요는 없지 않을까요?

```suggestion
	@PutMapping(
		value = "/me/profile-images",
		headers = API_VERSION_HEADER_NAME + "=" + 1,
	)
```

Reply:
consumes = MediaType.MULTIPART_FORM_DATA_VALUE 를 사용하는게 파일 업로드에 보장된 기본적인 방법이라고 생각해서 이 방법이 가장 나을 것 같다고 생각했습니다! 혹시 생각하시는 다른 방법이 있으실까요?